src/index.css

```diff
@@ -1,15 +1,20 @@
 body {
-  margin: 0;
   font-family: "SangBleu OG Serif Light Regular";
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  text-decoration: none;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
 }
 
 button {
-  margin: 0;
-  font-family: "SangBleu OG Serif Light Regular";
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  cursor: pointer;
+  font: "favorit";
```

Comment:
Ah yes. this is the one, "font-family"

Reply:
Could it be `font-family: favorit, serif;`?
 That way system falls back to a serif font if favorit not available. Also, i feel like if we are to use the font "favorit", we need to import the font?

Reply:
https://www.abcdinamo.com/typefaces/favorit

Reply:
Will ask the client to purchase

Reply:
Update. Advised client to scrap `Favorit` and move forward with `Helvetica Neue` as it's very similar **and** free.

Reply:
Just reading this now! My comment your other PR was having not seen this